src/game/objects/arena_constraints.h

```diff
@@ -1,8 +1,8 @@
 #ifndef ARENA_CONSTRAINTS_H
 #define ARENA_CONSTRAINTS_H
 
-#define ARENA_LEFT_WALL 20
-#define ARENA_RIGHT_WALL 300
-#define ARENA_FLOOR 190
+#define ARENA_LEFT_WALL 5120
+#define ARENA_RIGHT_WALL 76800
+#define ARENA_FLOOR 48640
```

Comment:
Instead of rolling magic values, could these instead be the operation ? e.g. 

#define ARENA_LEFT_WALL (20 * 256)

Compiler will probably optimize that anyway, and it would be much clearer on what these values are.

Reply:
That's why I suggested a macro so we could do something like `#define ARENA_FLOOR  PSEUDOFLOAT(190)`